### ISSUE TITLE
View doesn't update after http call during prerender

### ISSUE THREAD

**Issue by josiah1888** [angular/universal-starter#158]
If I make an http call, the server will make the call as expected, but not update the view during serialization.

```
@Component({
  selector: 'app',
  template: 'Hello Universal App Data: {{data}}'
})
export class App {
  public data: string = 'initial string';
  constructor(private http: Http) { }

  ngOnInit() {
    this.http.get('http://jsonplaceholder.typicode.com/posts/1')
      .subscribe(i => {
        const title = i.json().title;
        console.log('I did it, found this: ', title)
        this.data = title;
      });
  }
}
```

In this example, we see the console output from the node server with the correct json, but `this.data` does not get updated in the view.

As soon as the client rerenders the page, it gets updated correctly but this will not help the initial paint.

Full example: https://github.com/angular/universal-starter/compare/master...josiah1888:http-example?expand=1

**Comment by gdi2290** [comment]
remove `import { HttpModule } from '@angular/http';` on the server and client since it's included in UniversalModule

**Comment by gdi2290** [comment]
reusing the server response isn't in yet but everything is ready to add it in the next minor release

**Comment by josiah1888** [comment]
Thanks @gdi2290 !

Just a thought,
I'm assuming there's a way to programmatically check if a module has been imported twice. It would be really nice if we got a warning message to the console telling us that we imported something more than once, especially when the problem is masked like in this situation where Http is encapsulated in another module.

Either way, that's out of the scope for this issue. Thanks again
